webtask.json

```
@@ -1,8 +1,8 @@
 {
   "title": "Auth0 Account Link",
   "name": "auth0-account-link",
-  "version": "2.6.6",
-  "preVersion": "2.6.5",
+  "version": "2.6.7",
```

since this isn't a runtime upgrade, a patch version bump should suffice.

---

Functional changes are being made, I think a minor version bump make sense:

```suggestion
  "version": "2.7.0",
```

Imho major and patch version bumps are the exceptional cases

---

Does seem like patch version bumps are being used aggressively in this repo tho 🤔

---

Well the node upgrade will be a major version bump? So I can switch it to a minor with the node upgrade coming soon.

---

Just swapped it for a minor version bump. `2.7.0`